mzbench/scenarios/fan_out.bdl

```diff
@@ -0,0 +1,79 @@
+#!benchDL
+
+
+#######
+# Broadcast Scenario:
+# 10k subscribers reading from the same topic "channels/<MF_CHANNEL_ID>/messages"
+# 1 publisher sending 1 msg/s to topic "channels/<MF_CHANNEL_ID>/messages"
+# Overall Msg rate: 1k msg/s
+# Message Size: 100 random bytes
+# Runtime: 5 min
+# QoS level: 2
+# NOTE: Update values in defaults or set environment variables according to your values
+# <MF_MQTT_ENPOINT> = IP/domain of your MQTT endpoint
+# <MF_THING_1_ID> = First thing ID of pre-provisioned Mainflux thing
```

Comment:
Should this be maybe `MQTT_THING_1_ID`?

Reply:
I would prefer to stay `MF_THING_1_ID` like in this comment block and to rename environment variable `MQTT_THING_1_ID` in scenario to it. And all others I mistakenly named `MQTT_THING...` instead of `MF_THING...`

Reply:
I agree, all out ENV vars start with `MF_` prefix.